### PR TITLE
Branch 192461382

### PR DESCRIPTION
**Pull request by qlzh727**
Manually merged following files:

	tensorflow/compiler/xla/tests/slice_test.cc
	tensorflow/java/src/gen/cc/source_writer.h
	tensorflow/python/keras/_impl/keras/integration_test.py

### CODE REVIEW

**Comment by qlzh727** [comment]
All tests passed. Waiting for owner approval.

**Review by yifeif — APPROVED** [review]

**Pull request merged**
